opentelemetry-sdk/tests/logs/test_handler.py

```diff
@@ -209,13 +211,56 @@ def test_log_record_trace_correlation(self):
             self.assertEqual(log_record.span_id, span_context.span_id)
             self.assertEqual(log_record.trace_flags, span_context.trace_flags)
 
+    def test_log_record_args_are_translated(self):
+        processor, logger = set_up_test_logging(logging.INFO)
+
+        with self.assertLogs(level=logging.INFO):
+            logger.info("Test message")
+
+        log_record = processor.get_log_record(0)
+        self.assertEqual(
+            set(log_record.attributes),
+            {
+                "thread.id",
+                "code.filepath",
+                "code.lineno",
+                "code.function",
+                "thread.name",
+            },
+        )
+
+    def test_format_is_called(self):
+        processor, logger = set_up_test_logging(
+            logging.INFO,
+            logging.Formatter("%(name)s - %(levelname)s - %(message)s")
+        )
+
+        with self.assertLogs(level=logging.INFO):
+            logger.info("Test message")
+
+        log_record = processor.get_log_record(0)
+        self.assertEqual(
+            log_record.body, "foo - INFO - Test message"
+        )
+
```

Comment:
Could you add a test for the modulo case such as `logging.info("foo %s %s", "bar", "baz")`
```suggestion


    def test_format_modulo(self):
        processor, logger = set_up_test_logging(
            logging.INFO,
            logging.Formatter("%(name)s - %(levelname)s - %(message)s")
        )

        with self.assertLogs(level=logging.INFO):
            logger.info("Test message %s %d", bar, 3.1)

        log_record = processor.get_log_record(0)
        self.assertEqual(
            log_record.body, "foo - INFO - Test message bar 3.1"
        )
```